Neos.Fusion/Classes/Core/ObjectTreeParser/Lexer.php

```diff
@@ -44,12 +44,14 @@ class Lexer
         Token::MULTILINE_COMMENT => <<<'REGEX'
         `^
           /\*               # start of a comment '/*'
-          [^*]*             # match everything until special case '*'
+          [^*]*             # consume until special case '*'
+          \*+               # consume all '*'
           (?:
-            \*[^/]          # if after the '*' there is a '/' break, else continue
-            [^*]*           # until the special case '*' is encountered - unrolled loop following Jeffrey Friedl
+            [^/*]           # break if its the end: '/'
```

Comment:
im not sure why `*` is excluded in the character set. It works, and all tests pass without the exclusion. And there is no performance dropoff.

Reply:
Im nearly sure that `[^/*]` is must be enough and that `*` can never be encountered here.

because before we consume all `*` so there cant be one and also as last operation in the un rolled loop we consume all `*`. So independent of which entry point is used, the next char cannot be a `*`.

Reply:
Do we have tests for various variants of fusion file includes with glob patterns?

Reply:
yes but this is about multiline comments. And this regex will do but i was just eager to understand everypart and think there is redundancy but nothing critical.